sig-testing/flakiness-sla.md

```diff
@@ -0,0 +1,40 @@
+# Kubernetes Testing Flakiness SLA
+
+This document captures the expectations of the community about flakiness in
+our tests and our test infrastructure. It sets out an SLA (Service Level 
+Agreement) for flakiness in our tests, as well as actions that we will
+take when we are out of SLA.
+
+
+## Definition of a "Flake"
+
+We'll start by the definition of a _flake_. A test result is considered to
```

Comment:
environment fails to reach preconditions in time = deploy flake (ping sig-$provider)
environment that satisfies preconditions = test flake (ping PR author)

IMO we should differentiate

eg: if I were the author of an Azure feature, should I be blocked on flakiness because the GKE tests were flaking?